CMakeLists.txt

```diff
@@ -344,8 +344,8 @@ endif(WITH_DPDK)
 
 option(WITH_BLKIN "Use blkin to emit LTTng tracepoints for Zipkin" OFF)
 if(WITH_BLKIN)
-  find_package(blkin REQUIRED)
-  include_directories(${BLKIN_INCLUDE_DIR})
+  set(BLKIN_LIBRARIES blkin lttng-ust)
+  include_directories(src/blkin/blkin-lib)
```

Comment:
can we add blkin as an external project, so its targets won't be populated into ceph's cmake targets, and "make install" won't include the blkin?